CHANGELOG.md

```diff
@@ -1,3 +1,70 @@
+## v1.5.0 (18/08/2020)
+
+**olp-sdk-core**
+
+- Added the new `@here/olp-sdk-core` module v1.0.0 and moved common components to it.
+
+**olp-sdk-authentication**
+
+- Added `@here/olp-sdk-core` as a dependency.
+- Updated `olp-sdk-fetch` to version 1.5.0.
+- Deprecated the `lib.version.ts` file. It will be removed by 02.2021. Use the same the from `@here/olp-sdk-core` package instead.
+- Deprecated the `./lib/HttpError.ts` file. It will be removed by 02.2021. Use the same the from `@here/olp-sdk-core` package instead.
```

Comment:
```suggestion
- Deprecated the `./lib/HttpError.ts` file. It will be removed by 02.2021. Use the same the from the `@here/olp-sdk-core` package instead.
```